targets/unity-v2/source/Shared/External/tpl35/System.Collections.Generic/DefaultEqualityComparer.cs

```diff
@@ -54,7 +54,9 @@ public int GetHashCode(object obj)
             return GetHashCode ((T)obj);
         }
 
+#pragma warning disable 108,114
```

Comment:
Can you please leave comments about which warnings are being disabled here?
It saves me the time of doing an internet search every time.